projects/packages/videopress/src/client/state/reducers.js

```diff
@@ -563,11 +565,49 @@ const purchases = ( state, action ) => {
 	}
 };
 
+const playbackTokens = ( state, action ) => {
+	switch ( action.type ) {
+		case SET_IS_FETCHING_PLAYBACK_TOKEN: {
+			return {
+				...state,
+				isFetching: action.isFetching,
+			};
+		}
```

Comment:
When adding the token fetch to the main dashboard libraries this will need to be set for each video

Reply:
Makes sense, so each one has it's own fetching state.